azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -64,6 +64,8 @@
   private static final Logger logger = Logger.getLogger(ExecutorManager.class);
   private final RunningExecutions runningExecutions;
   private final RunningExecutionsUpdaterThread updaterThread;
+  private final int maxConcurrentRunsOneFlow;
```

Comment:
The comment on top of the class says this is deprecated. Why do we need to modify this class?

Reply:
Just removing unused variables. There was a lot of duplicate code and I fixed it in other PR for ExecutionController and ExecutorManager. These variables are unused here now.